composer.json

```diff
@@ -55,7 +55,7 @@
         "symfony/filesystem": "^3.0",
         "symfony/finder": "^3.0",
         "symfony/form": "^3.0",
-        "symfony/framework-bundle": "^3.0",
+        "symfony/framework-bundle": "^3.3",
```

Comment:
The manifest based versioning is only available on https://symfony.com/blog/new-in-symfony-3-3-manifest-based-asset-versioning /cc @danrot @chirimoya